starfinder/db/models.py

```diff
@@ -164,44 +193,119 @@ class Equipment(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=False)
+    price = db_engine.Column(db_engine.Integer(), nullable=False)
+    range_grenade = db_engine.Column(db_engine.String(64), nullable=False)
+    capacity = db_engine.Column(db_engine.String(64), nullable=False)
+    bulk = db_engine.Column(db_engine.String(64), nullable=False)
+    special = db_engine.Column(db_engine.String(64), nullable=False)
+
+
+class MeleeWeapon(db_engine.Model, ModelBase, HasId):
+    category = db_engine.Column(db_engine.Integer(), nullable=False)
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=True)
+    price = db_engine.Column(db_engine.Integer(), nullable=True)
+    damage = db_engine.Column(db_engine.String(64), nullable=False)
+    critical = db_engine.Column(db_engine.String(64), nullable=True)
+    bulk = db_engine.Column(db_engine.String(64), nullable=True)
+    special = db_engine.Column(db_engine.String(64), nullable=False)
+    powered = db_engine.Column(db_engine.Binary(), nullable=False)
+    operative = db_engine.Column(db_engine.Binary(), nullable=False)
+    description = db_engine.Column(db_engine.String(64), nullable=False)
+    
+
+class WeaponCategory(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
+    type_wep = db_engine.Column(db_engine.String(64), nullable=False)
+    hands = db_engine.Column(db_engine.Integer(), nullable=False)
+    category = db_engine.Column(db_engine.String(64), nullable=False)
+
+class SolarianCrystal(db_engine.Model, ModelBase, HasId):
+    level = db_engine.Column(db_engine.Integer(), nullable=False)
+    price = db_engine.Column(db_engine.Integer(), nullable=False)
+    damage = db_engine.Column(db_engine.String(64), nullable=False)
+    critical = db_engine.Column(db_engine.String(64), nullable=False)
+    description = db_engine.Column(db_engine.String(64), nullable=False)
+
+
+class OtherEquipment(db_engine.Model, ModelBase, HasId):
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    type_wep = db_engine.Column(db_engine.String(64), nullable=False)
+    price = db_engine.Column(db_engine.Integer(), nullable=False)
+    bulk = db_engine.Column(db_engine.String(64), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=True)
+
+
+class RangedWeapon(db_engine.Model, ModelBase, HasId):
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    category = db_engine.Column(db_engine.Integer(), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=False)
+    price = db_engine.Column(db_engine.Integer(), nullable=False)
+    damage = db_engine.Column(db_engine.String(64), nullable=False)
+    range_atk = db_engine.Column(db_engine.String(64), nullable=False)
```

Comment:
How about `ranged_attack` or `attack_range`? Whichever is more accurate for this field's contents.